src/org/opendatakit/briefcase/model/BriefcaseAnalytics.java

```diff
@@ -27,15 +27,17 @@ public BriefcaseAnalytics() {
     }
```

Comment:
If you took my suggested changes verbatim right from https://github.com/dcbriccetti/briefcase/tree/PR177-improvements then I can approve this very quickly. Did you?